### ISSUE TITLE
[Bug]: Filter news by site partners

### ISSUE THREAD

**Issue by katjam** [geeksforsocialchange/the-trans-dimension#456]
## Description
Currently we do include the partner in News data but I don't think we filter out News Items that belong to partners outside the partnership.

This can be handled in `src/Data/Placecal/Articles.elm` because we have access to the partner list here.

**Comment by kimadactyl** [comment]
I think its OK to just show everything for now?

**Comment by katjam** [comment]
Agree. It will only be a problem if people start using it. I think the name resolution from the Partners authoring the story won't work if the partners are not included in partnership tags for Trans Dimension but probably will not be noticeable because "I think" they will show as an ID number. We might want to check this though, in case it breaks more dramatically than that.

![image](https://github.com/user-attachments/assets/a78ba9c1-705c-4a2d-89ff-8f3e457bceaf)
